api/document/js/fetch-doc.ts

```diff
@@ -9,6 +9,15 @@ export function convertNode(node) {
 }
 
 const NODE_TYPE_CONVERTERS = {
+  heading(node) {
+    // Duplicates logic in `ui`
+    const depth = node.identifier
+      .split('_')
+      .filter(e => e === 'sec')
+      .length + 1;
+    const text = (node.text || '').replace(/\s+/g, ' ');
+    return schema.nodes.heading.create({ depth }, schema.text(text));
```

Comment:
Do you think we should deal with the potential for the depth to be greater than 6?  We could, for instance, use [`aria-level`](https://www.w3.org/WAI/GL/wiki/Using_role%3D%22heading%22_to_identify_headings#Example_2) in that case... ah well, probably something you've already thought about w/ the viewer, and probably pretty unlikely anyways.

Reply:
That's a good question; we'd need to check in with design, too, as I'm sure they'd have an opinion. I _really_ like the use-h1s-for-everything option as a no-thinking solution; do you think that's supported enough for us to use in the public-facing ui?

Reply:
Good question, I just addressed this in #896!